src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -2,31 +2,9 @@
   // Component variables
   --pf-c-about-modal-box--BackgroundColor: var(--pf-global--palette--black-1000); // Modal uses a non-standard background color
   --pf-c-about-modal-box--BoxShadow: 0 0 100px 0 rgba(255, 255, 255, .05);  // This is a special one-off glow for the about modal
```

Comment:
@mceledonia do you see any reason to keep this box shadow on the about modal around? it's a 100px spread, white w/ 50% opacity shadow. It's barely visible, and if we're moving the the about modal into the modal component, the modal has its own box shadow.

The current about modal w/ 100px shadow

<img width="1901" alt="Screen Shot 2022-11-07 at 5 10 32 PM" src="https://user-images.githubusercontent.com/35148959/200435043-66852198-2026-42de-b0d9-3230955022c1.png">

No shadow, just to compare

<img width="1945" alt="Screen Shot 2022-11-07 at 5 10 36 PM" src="https://user-images.githubusercontent.com/35148959/200435082-d746f0fc-928e-4c24-8359-87af6e37ee6e.png">

using the modal component's default shadow

<img width="1967" alt="Screen Shot 2022-11-07 at 5 11 25 PM" src="https://user-images.githubusercontent.com/35148959/200435132-a3eb83d5-24ea-4130-8ce0-af678ef25fb6.png">

Reply:
Nope good catch, let's drop it!